src/main/java/GeneralService/OnlineServiceHandler.java

```diff
@@ -0,0 +1,15 @@
+package GeneralService;
+
+import onlineservices.Service;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+import java.util.List;
+
+@org.springframework.stereotype.Service
+public class OnlineServiceHandler {
+    @Autowired
+    @Qualifier("activeServices")
+    private List<Service> services;
```

Comment:
if the name of the variable is the same as the name of the method from the config class, you do not need to use a qualifier, rename the variable to activatedServices and remove the qualifier annotation